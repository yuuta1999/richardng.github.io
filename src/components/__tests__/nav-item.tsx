import React from 'react'
import { render, cleanup } from '@testing-library/react'
import 'jest-styled-components'

import NavItem from '../nav-item'

afterEach(cleanup)

it('Navbar works', () => {
  const { container } = render(<NavItem />)

  expect(container.firstChild).toMatchSnapshot()
})
