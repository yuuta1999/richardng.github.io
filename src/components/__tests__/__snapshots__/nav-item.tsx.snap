// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar works 1`] = `
.c0 {
  padding-right: 0.5rem;
}

<li
  class="c0"
/>
`;
